### DISCUSSION TITLE
openwrt挂载alist启动后进入WEB界面输入账号和密码之后显示登录成功，但是又显示Token is expired，进不到管理界面

### DISCUSSION THREAD

**Discussion by lzp544765181** [AlistGo/alist#3007] [Q&A]
![微信截图_20230111215955](https://user-images.githubusercontent.com/122459119/211825710-2effb160-121e-46ea-9f90-984e1f1e73ba.png)

**Comment by NovaShen0724** [comment]
无痕进去看看？

**Comment by 235buluo** [comment]
每天都会遇到这个问题，太烦人了

**Comment by Maxhit110** [comment]
我也是这样，重新安装也不行。。真不知道怎么处理。。

**Comment by wanyunr** [comment]
> 无痕进去看看？

我也遇到同样的问题，登录成功后又立刻显示`Token is expired`，无痕浏览也没用。
重启Alist服务端似乎解决了
